proto/status.proto

```diff
@@ -27,6 +26,33 @@ option (gogoproto.sizer_all) = true;
 option (gogoproto.marshaler_all) = true;
 option (gogoproto.unmarshaler_all) = true;
 
+// MVCCStats tracks byte and instance counts for:
```

Comment:
This doesn't really belong here. It belongs in mvcc.proto. Is there some reason we're not putting it there? Is it impossible to include mvcc.proto in status.proto?